Backpack-SwiftUI/Card/Classes/BPKDividedCard.swift

```diff
@@ -19,20 +19,23 @@
 import SwiftUI
 
 public struct BPKDividedCard<PrimaryContent: View, SecondaryContent: View>: View {
+    private let isElevated: Bool
     private let primaryContent: PrimaryContent
     private let secondaryContent: SecondaryContent
     private var tapAction: () -> Void = {}
 
     public init(
+        isElevated: Bool = true,
         @ViewBuilder primaryContent: () -> PrimaryContent,
         @ViewBuilder secondaryContent: () -> SecondaryContent
     ) {
+        self.isElevated = isElevated
```

Comment:
Can you please use the same Enum for the elevation option? 
We can basically 'copy' the way BPKCard does it.

Reply:
Can we just use `BPKCard.Elevation` directly? If we define an elevation Enum inside `BPKDividedCard`, we have to convert `BPKDividedCard.Elevation` to `BPKCard.Elevation`.

Reply:
Yes we can use it directly.